Breeding-Insight/bi-api
# BI-1665 Store observations when importing new environments

# Description
[BI-1665](https://breedinginsight.atlassian.net/browse/BI-1665)

# Dependencies
sgn:  feture/BI-1665 branch
bi-web: develop branch

# Set Up
in your _sgn_local.conf_ add the key-value pair:
`brapi_observations_require_login 0`

# Testing

1. Import an Experiments & Observation file with at least one phenotype column with the corresponding timestamp column(s).
2. Confirm the import.
3. Insure that the data has been saved in he backing BrAPI server  

# Checklist:



[BI-1665]: https://breedinginsight.atlassian.net/browse/BI-1665?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
